R/reactives.R

```diff
@@ -461,6 +682,7 @@ Observable <- R6Class(
   'Observable',
   portable = FALSE,
   public = list(
+    .origFunc = 'function',
```

Comment:
Is the only purpose of storing `.origFunc` is so that the body can be deparsed in the `format.reactiveExpr` method?